src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1766,11 +1766,17 @@ internal static string GetTokString(TokKind kind)
             }
         }
 
-        public static string Format(string text)
+        /// <summary>
+        /// Formats/Pretty Print the given expression text to more a readable form.
+        /// </summary>
+        /// <param name="text">Expression text to format.</param>
+        /// <param name="flags">Optional flags to customize the behavior of underlying lexer and parser. By default, expression chaining is enabled.</param>
+        /// <returns>Formatted expression text.</returns>
+        public static string Format(string text, Flags flags = Flags.EnableExpressionChaining)
         {
             var result = ParseScript(
                 text,
-                flags: Flags.EnableExpressionChaining);

```

Comment:
Dropping the comment to prevent auto completion until i test this out with a private deployment